### PR TITLE
changes pytest to work with RLTest framework

### PR DESCRIPTION
**Pull request by MeirShpilraien**

### CODE REVIEW

File: src/pytest/CMakeLists.txt [hunk 1]
```diff
@@ -4,14 +4,10 @@ FILE(GLOB PY_TEST_FILES "test*.py")
 FOREACH(n ${PY_TEST_FILES})
     GET_FILENAME_COMPONENT(test_name ${n} NAME_WE)
     ADD_TEST(NAME "PY_${test_name}"
-        COMMAND "python" "-m" "unittest" "${test_name}"
+        COMMAND "RLTest" "--module" "$<TARGET_FILE:redisearch>" "--tests-file" "${test_name}" "--clear-logs"
```
Comment (mnunberg) [inline src/pytest/CMakeLists.txt]: please change command name to normal naming conventions (i.e. all lower case)

**Comment by MeirShpilraien** [comment]
Regarding config file I will postponed it for later as we are already behind schedule and I do not think its mandatory.
Improve output, I like it as it is, if you have another suggestion pls let me know and we discuss it.
Regarding naming, do you want RLTest to be changed to rltest? thats all?

**Comment by mnunberg** [comment]
I rely on the config file in rmtest for my own execution of tests. I’ll go ahead and add the feature myself when I have a chance to actually play with it (later today…). I’ll also go ahead and revise the naming in RLTest per Python naming conventions (as we discussed earlier) and make a PR to that effect.

**Comment by mnunberg** [comment]
I tried adding some features to RLTest, but without much success. I don't want my current workflow (wrt Valgrind and GDB) to be disrupted. All I ask for is:

1. Config file format with comments (you could use JSON, then scan line-for-line, omitting those lines optionally with leading whitespace and begin with a `#`)
2. Ability to set the debugger via the environment (without specifically enabling valgrind). This would allow to do something like
```
REDIS_DEBUGGER="gdb --args" rltest ....
```
3. Redis output itself is captured. While this might be desirable when all the tests pass, it is crucial to have the output when the tests fail. There is no such option to output to the screen.

**Comment by MeirShpilraien** [comment]
1.  I will try to move to ini format.
2.  you can do --debug, then the env will be set up but the test will not start till you press a button, this allows you to attach with gdb to any process you like. We can use -vv to see the processes pid and attach to them.
3. Redis output is capture to a file (file names contains the test name so its very easy to find), you want the output to be printed to the screen in case of a failure? I think it will garbage the output and make it difficult to read.

**Comment by MeirShpilraien** [comment]
@mnunberg I drop the ini idea because it not possible to specify values types and in this format and I did not wanted to start type check (and parse) for each value by myself.
Though I did find a quick way to handle you requirement using a built in capability in the args library I am using.
The framework now searches for default config file called config.txt.
The format is the same as the command line argument and its possible to comment lines from the config file.

Have fun :)

**Pull request merged**